docs-conceptual/azurermps-5.7.0/other-install.md

```diff
@@ -1,104 +1,54 @@
-
-> [!NOTE]
-> If you have previously installed Azure modules from the PowerShell Gallery, the installer
-> automatically removes them. This simplifies your environment by ensuring that only one version
-> of Azure PowerShell is installed. However, there are scenarios where you may need multiple
-> versions installed at the same time.
->
-> PowerShell Gallery modules install modules in
-> `$env:ProgramFiles\WindowsPowerShell\Modules`. In contrast, the WebPI installer
-> installs the Azure modules in `$env:ProgramFiles(x86)\Microsoft SDKs\Azure\PowerShell\`.
->
-> If an error occurs during install, you can manually remove the `Azure*` folders in your
-> `$env:ProgramFiles\WindowsPowerShell\Modules` folder, and try the installation again.
+To run Azure PowerShell in a Docker container, see [Run Azure PowerShell in Docker](azurerm-ps-in-docker.md).
 
-Once the installation completes, your `$env:PSModulePath` setting should include the directories
-containing the Azure PowerShell cmdlets. The following command can be used to verify that the Azure
-PowerShell is installed properly:
-
-```powershell
-# To make sure the Azure PowerShell module is available after you install
-Get-Module -ListAvailable Azure* | Select-Object Name, Version, Path
-```
-
-> [!NOTE]
-> There is a known issue that can occur when installing from WebPI. If your computer requires a
-restart due to system updates or other installations, it may cause updates to `$env:PSModulePath` to
-fail to include the path where Azure PowerShell is installed.
+To install on Linux or macOS environments, see [Install Azure PowerShell on macOS or Linux](install-azurermps-maclinux.md).
 
-When attempting to load or execute cmdlets after installation, you can receive the following error
-message:
+## Install or update on Windows using the MSI Package
 
-```
-PS C:\> Connect-AzureRmAccount
-Connect-AzureRmAccount : The term 'Connect-AzureRmAccount' is not recognized as the name of a cmdlet,
-function, script file, or operable program. Check the spelling of the name, or if a path was
-included, verify that the path is correct and try again.
-At line:1 char:1
-+ Connect-AzureRmAccount
-+ ~~~~~~~~~~~~~~~~~~~~~~~
-    + CategoryInfo          : ObjectNotFound: (Connect-AzureRmAccount:String) [], CommandNotFoundException
-    + FullyQualifiedErrorId : CommandNotFoundException
-```
+Azure PowerShell can be installed using the MSI file available from
+[GitHub](https://github.com/Azure/azure-powershell/releases/tag/v5.7.0-April2018). If you have installed previous
+versions of Azure modules as an MSI, the installer automatically removes them. The MSI package installs
+modules in `${env:ProgramFiles}\WindowsPowerShell\Modules`.
 
-This error can be corrected by restarting the machine or importing the module using the fully
-qualified path.
+When you install with the MSI, the `AzureRM` and `Azure` modules are pre-loaded into your PowerShell sessions.
```

Comment:
If this is true, then should the instructions in `install-azurerm-ps.md` about setting up a profile to automatically import also be altered? Is the canonical way in PowerShell to load modules through adding `Import-Module` to a profile, or to set `PSModuleAutoLoadingPreference`?

Reply:
No, that statement still stands. The point is that you should always use Import-Module and not rely on the autoloading preference, because the autoloading preference may be set to not allow autoloading.

Reply:
For Line 30, a better statement would be:

The MSI package installs bothe the `AzureRM` and `Azure` modules.